<launch>
<arg name="to_cpu" default="False"/>
    <node name="graphce_detector" pkg="object_detector" type="graphce_detector.py" output="screen"/>
        <param name="to_cpu" value="$(arg to_cpu)"/>
</launch>
